docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -91,27 +95,60 @@ Install the iOS SDK via Swift Package Manager (SPM). There are two installation
 
 Add the following dependency to Package.swift:
 
-``` javascript
+```js
 dependencies: [
-.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.3.0"),
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "1.2.0"),
 ]
 ```
 
 ### Installing with Xcode
 
-Add the following dependency via Xcode:
+Add the following entry in the Package Dependencies for your apps:
 
-![Xcode Dependency](images/xcode-uid2-ios-sdk.png)
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-sdk | git@github.com:IABTechLab/uid2-ios-sdk.git | Up to next major version: 1.2.0 < 2.0.0 |
 
 ## Usage Guidelines
 
-The initial UID2 Identity *must* be generated by the implementing application and then passed into the UID2 SDK. Because of security requirements, the UID2 Mobile SDKs cannot create new UID2 identities.
+The **UID2Manager** singleton is the primary developer API for the UID2 SDK for iOS. It is responsible for storing, refreshing, and retrieving UID2 Identity and according token.
+
+For iOS, the `UID2Manager` is initialized automatically the first time it is accessed. You can configure it to support automatic or manual refresh capabilities.
+
+There are two ways to establish an initial UID2 Identity:
+
+1. Generate the UID2 identity using DII&#8212;email (hashed) or phone number (hashed). For integration instructions, see [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md).
```

Comment:
"hashed or unhashed"?

Reply:
@mcollins-ttd : this question, I cannot answer. @sunnywu help with this please?

Reply:
Changed to hashed or unhashed, both instances -- thx.